src/index.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable array-callback-return */
+/* eslint-disable no-undef */
+/* All Functions */
+import './style.css';
+import editTasks from './js.modules/editFunction.js';
+import removeTasks from './js.modules/removeFunction.js';
+import updatingData from './js.modules/updatingData';
+import removeAllTasks from './js.modules/removeALlfunction';
+
+/* All HTML References */
+const taskInput = document.querySelector('input');
+const tasksContainer = document.querySelector('.tasks-container');
+const completedTasks = document.querySelector('button');
```

Comment:
Please, kindly try to avoid creating global variables, as it is a good JS practice.